great-idea-website/css/index.css

```diff
@@ -63,4 +63,125 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+#container {
+	width: 60%;
+	margin: 0 auto;
+	padding-left: 5%;
+}
+
+#navwrapper {
+	text-align: center;
+}
+
+nav {
+	display: inline-block;
+	vertical-align: top;
+	margin-top: 35px;
+}
+
+nav > a {
+	color: gray;
+	margin-right: 65px;
+	text-decoration: none;
+	font-size: .9em;
+}
+
+nav > a:hover {
+	color: royalblue;
+	border-bottom: 1px solid royalblue;
+}
+
+.logo {
+	margin-top: 25px;
+}
+
+h1 {
+	font-size: 4.5em;
+	width: 250px;
+	vertical-align: top;
+	margin-left: 55px;
+	letter-spacing: 2px;
+}
+
+#banner {
+	text-align: center;
+	margin-top: 50px;
+	display: inline-block;
+	vertical-align: top;
+}
+
+#container > header > div:nth-child(2) > img {
+	padding-left: 250px;
+}
+
+button {
+	border: 1px solid black;
+	font-family: "Titillium Web";
+	background-color: white;
+	text-align: center;
+	width: 160px;
+	height: 35px;
+	position: relative;
+	margin-left: 55px;
+	font-weight: bold;
+	font-size: 1em;
+}
+
+button:hover {
+	border: 1px solid royalblue;
+	text-decoration: underline;
+	color: royalblue;
+}
+
+#bannerwrapper {
+	margin-top: 50px;
+	border-bottom: 1px solid black;
+	padding-bottom: 60px;
+}
+
+.dblbox {
+	display: inline-block;
+	width: 49%;
+	text-align: left;
+	margin-left: 7px;
+}
+
+.trplbox {
+	display: inline-block;
+	width: 32%;
+	margin-left: 7px;
+	text-align: left;
+}
+
+section {
+	text-align: center;
+}
+
+h2 {
+	padding-top: 35px;
+}
+
+.middle-img {
+	margin-left: 60px;
+	margin-top: 50px;
+}
+
+#btmsection {
+	padding-bottom: 55px;
+	border-bottom: 1px solid black;
+}
+
+#phone {
+	margin-top: 15px;
+	margin-bottom: 15px;
+}
+
+#copy {
+	font-style: italic;
+	text-align: center;
+	margin-top: 15px;
+	margin-bottom: 30px;
+	font-family: 'Shadows Into Light';
```

Comment:
I would like to see more specificity around these areas of your code. There are too many styles be selected with id's rather than with classes. I'd's are generally only used in the cases where you might need to use javascript to grab an item so you want it to stand out from the rest. Classes should always be your first go too.. *hint* you won't use id's very often if at all in react so try getting comfortable with classes instead.